Gemfile

```diff
@@ -5,5 +5,6 @@ gem 'asciidoctor', '~> 2.0', '>= 2.0.20'
 # Uncomment for ability to convert Markdown to AsciiDoc
 gem 'kramdown-asciidoc'
 
-# Needed for https://github.com/ruby/rexml/issues/131#issuecomment-2116267578
-gem 'rexml', '= 3.2.6'
+# required by https://www.ruby-lang.org/en/news/2024/10/28/redos-rexml-cve-2024-49761/
+# this resolved 5 other cves fixed since 3.2.6 so we should not explcitly pin.
+gem "rexml", ">= 3.3.9"
```

Comment:
i think we might also just be able to remove this 

based on https://github.com/openstack-k8s-operators/openstack-operator/blob/main/Gemfile

Reply:
this is only used to build the docs right? We could try removing it and see if the job is still working

Reply:
i think it will get pulled by the package that actually using it so ill try removing it and see if it also works.

and yes just used for the docs.

Reply:
done